btcec/schnorr/musig2/sign.go

```diff
@@ -697,3 +697,12 @@ func CombineSigs(combinedNonce *btcec.PublicKey,
 
 	return schnorr.NewSignature(&nonceJ.X, &combinedSig)
 }
+
+// GetGenerator returns a public key at the Generator Point.
+func GetGenerator() *secp.PublicKey {
```

Comment:
How about we move this to just the top level `btcec` package? There it can be `Generator`, so the call path would be: `btcec.Generator()`.